c++/src/H5Group.cpp

```diff
@@ -274,4 +274,15 @@ Group::~Group()
     }
 }
 
+//--------------------------------------------------------------------------
+// Function:    Copy assignment operator
+Group &
+Group::operator=(const Group &original)
+{
+    if (&original != this) {
+    }
```

Comment:
@seanm, I would do the same here as Attribute::operator=.

Reply:
Done.

Reply:
Thanks!